config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -211,6 +211,7 @@ presubmits:
         - --test-cmd-args=cluster-loader2
         - --test-cmd-args=--nodes=500
         - --test-cmd-args=--provider=kubemark
+        - --env=CL2_ENABLE_PVS=false # TODO(https://github.com/kubernetes/perf-tests/issues/803): Fix me
```

Comment:
Sorry, it might have been not clear. Now we have this env, let's delete the lines 225, 226, 419,420. 
```
        # TODO(https://github.com/kubernetes/perf-tests/issues/803): Re-enable
        # - --test-cmd-args=--testoverrides=./testing/load/experimental/overrides/enable_pvs.yaml
```
They are redundant.

Also, coud you squash the commits into one? Thanks!